RecoTracker/TkDetLayers/src/TBLayer.h

```diff
@@ -25,14 +25,14 @@ class TBLayer: public BarrelDetLayer {
 
   // GeometricSearchDet interface
   
-  virtual const std::vector<const GeomDet*>& basicComponents() const final {return theBasicComps;}
+  virtual const std::vector<const GeomDet*>& basicComponents() const {return theBasicComps;}
```

Comment:
this one can't be final because there is an overload in the derived Phase2OTtiltedBarrelLayer.
so, unless that overload can be removed ...